Simplenote/build.gradle

```
@@ -150,7 +150,9 @@ android.applicationVariants.all { variant ->
             key == "LOGIN_EMAIL" ||
             key == "LOGIN_PASSWORD" ||
             key == "SENTRY_DSN" ||
-            key == "WPCOM_CLIENT_ID") {
+            key == "WPCOM_CLIENT_ID" ||
+            key == "SCREENSHOT_USER_EMAIL" ||
+            key == "SCREENSHOT_USER_PASSWORD") {
```

```suggestion
            key == "SCREENSHOT_EMAIL" ||
            key == "SCREENSHOT_PASSWORD") {
```

Is `_USER` necessary?  We could remove it to make the variables `SCREENSHOT_EMAIL` and `SCREENSHOT_PASSWORD` to mimic the `LOGIN_EMAIL` and `LOGIN_PASSWORD` variables.

---

Yeah, will do 👍 .